website/blog/2023-11-13-curious-ternaries.md

```diff
@@ -12,7 +12,7 @@ Please give the `--experimental-ternaries` option a try and let us know what you
 
 _For a quick tl;dr, [see the release post](/blog/2023/11/13/3.1.0)._
 
-<!-- truncate -->
+<!--truncate-->
```

Comment:
Does the one with space work? If it does, maybe we should use `<!-- truncate -->` instead, we always add space to js comments, why not html comments.